src/code/z_en_a_keep.c

```diff
@@ -1,27 +1,27 @@
 #include "global.h"
 
-void EnAObj_Init(ActorEnAObj* this, GlobalContext* globalCtx) {
-    ActorEnAObj* s0 = (ActorEnAObj*)this;
+void EnAObj_Init(EnAObj* thisx, GlobalContext* globalCtx) {
+    EnAObj* s0 = (EnAObj*)thisx;
     s0->base.textId = ((s0->base.params >> 8) & 0xFF) | 0x300;
     s0->base.params = (s0->base.params & 0xFF) - 9;
     Actor_ProcessInitChain((Actor*)s0, &enAObjInitVar);
     ActorShape_Init(&s0->base.shape, 0, (ActorShadowFunc)func_800B3FC0, 12);
     Collider_InitAndSetCylinder(globalCtx, &s0->collision, (Actor*)s0, &enAObjCylinderInit);
     Collider_UpdateCylinder((Actor*)s0, &s0->collision);
     s0->base.colChkInfo.mass = 255;
-    s0->update = (ActorFunc)EnAObj_Update1;
+    s0->actionFunc = (EnAObjActionFunc)EnAObj_Update1;
 }
 
-void EnAObj_Destroy(ActorEnAObj* this, GlobalContext* globalCtx) {
+void EnAObj_Destroy(EnAObj* this, GlobalContext* globalCtx) {
     ColliderCylinder* a2 = &this->collision;
     Collider_DestroyCylinder(globalCtx, a2);
 }
 
-void EnAObj_Update1(ActorEnAObj* this, GlobalContext* globalCtx) {
+void EnAObj_Update1(EnAObj* this, GlobalContext* globalCtx) {
     s16 v0;
     s32 v1;
     if (func_800B84D0((Actor*)this, globalCtx) != 0) {
```

Comment:
```suggestion
    if (func_800B84D0(&this->actor, globalCtx) != 0) {
```
Maybe a boolean, too?